## Results {.page_break_before}

#### 1. Enrichment of ASoC Variants

To validate our predicion model, we first performed enrichment analysis of allele-specific open-chromatin (ASoC) variants. Genetic variants prioritized by our prediction model are expected to have lagre functional effects. One way to test our hypothesis is calculating the enrichment of some genetic variants with konwn functional effects in our top predictions. ASoC variants are overrepresented in brain enhancers, transcription-factor-binding sites, and quantitative-trait-loci associated with gene expression, histone modification, and DNA methylation. We obtained ASoC variants in neural progenitor cells (NPC) and glutamatergic (iN-Glut) neurons from a neuron ATAC-seq study. We then acquired single nucleotide variants in open chromatin regions of NPC and iN-Glut by mapping against 1000 Genome and prioritized them by our Brain-ResNet scores. The top 10,000 predicted genetic variants show 4 fold enrichment of ASoC variants in NPC and iN-Glut. AS a comparsion, we also prioritized genetic variants with Functional significance (Funsig) score and CADD score. Funsig is a measure of the signficance of magnitude of predicted chromatin effect and evolutionary conservation and CADD score is a measure of the deleteriousness of genetic variants. As shown in fig1, our Brain-ResNet scoring outperforms Funsig and CADD scoring in terms of identifying function variants.

#### 2. Sign Consistentcy

Next, we applied our prediction model to NPC and iN-Glut ASoC variants and compared the observed allelic imbalance and the predicted difference in functional effects between reference and alternative alleles. Our prediction model tracks the observed allelic imbalance ratio with a correlation of 0.44 and 0.40. Notably, we found 70% variants show consistent sign in observed allelic imbalace and estimated effect, which demonstrates taht the prediction model accurately captures the direction of effect. 

#### 3. Evolutionary Constraint

Functionally important variants tend to be under evolutionary constraint. We calculated GERP score, which measures the number of substitutions "rejected" by evolutionary constraint, for top predicted variants and randomly sampled variants in 31 cell types. Higehr GERP score indicates greater magnitude of evolutionary constraint. As shown in fig3, for most cell types, our prediction model successfully prioritized genetic variants that are under higher evolutionary constraint and more likely to have siginificant biological functions. 

#### 4. Purifying Selection




## Figures

![
**ASoC Enrichment.**
Bar plot comparing the enrichment of allele specific open chromatin variants among three groups in two cell types.
](https://github.com/sq-96/resources/raw/master/ASoC%20Enrichment.png "Square image"){#fig:ASoC Enrichment}

![
**GERP Score Distribution.**
Bar plot comparing the evolutionary constraint between two groups in 31 cell types.
](https://github.com/sq-96/resources/raw/master/GERP%20Score.png "Square image"){#fig:GERP Score Distribution}

![
**Sign Consistency.**
Scatter plot comparing the observed allelic imbalance and the predicted difference in functional effects between reference and alternative alleles.
](https://github.com/sq-96/resources/raw/master/sign%20consistency.png "Square image"){#fig:Sign Consistency}

![
**Minor Allele Frequency.**
Scatter plot showing the negtive correlation between minor allele frequency and Brain-ResNet predicted scores.
](https://github.com/sq-96/resources/raw/master/NSC_CN_GA_DN.png "Square image"){#fig:Minor Allele Frequency}

![
**Heatmap.**
Heatmap showing functional effects of credible set SNPs in 31 cell types.
](https://github.com/sq-96/resources/raw/master/heatmap.png "Square image"){#fig:Heatmap}

