## Results {.page_break_before}

#### 1. Enrichment of ASoC Variants

To validate our predicion model, we first performed enrichment analysis of allele-specific open-chromatin (ASoC) variants. Genetic variants prioritized by our prediction model are expected to have lagre functional effects. We hypothesize that our predictions are enriched for genetic variants with some known functions. ASoC variants have been established to be functional in brain, impacting gene expresison, histone modification and DNA methylation. We obtained ASoC variants in neural progenitor cells (NPC) and glutamatergic (iN-Glut) neurons from a neuron ATAC-Seq study. We then acquired all single nucleotide variants in open chromatin regions of NPC and iN-Glut and prioritized them by our NPC and iN-Glut Brain-ResNet scores. The top 10,000 predicted genetic variants show 4 fold enrichment of ASoC variants in NPC and iN-Glut. To show the strength of our model, we also prioritized genetic variants within open chromatin regions by Functional significance (Funsig) score and CADD score. Funsig is a measure of the signficance of magnitude of predicted chromatin effect and evolutionary conservation, and CADD score is a measure of the deleteriousness of genetic variants. As shown in Fig1, our Brain-ResNet scoring significantly outperforms Funsig and CADD scoring. This gaining may arise from two apsects. First, our model uses functional genomic data from matched cell types, which could more accurately reveal the chromatin status. Second, our model uses ResNet architecture and is based on transfer learning, which could more precisely learn regulatory codes from DNA sequences. To further address the importance of matched cell types, we used Brain-ResNet scores from the other 30 cell types to prioritize genetic variants in NPC and iN-Glut. As shown in Fig2, top predictions prioritized by matched cell types generally have higher enrichment of ASoC variants.

#### 2. Sign Consistentcy

Next, we applied our prediction model to NPC and iN-Glut ASoC variants and compared the observed allelic imbalance and the predicted difference in functional effects between reference and alternative alleles. Our prediction model tracks the observed allelic imbalance ratio with a correlation of 0.44 and 0.40. Notably, we found 70% variants show consistent sign in observed allelic imbalace and estimated effect, which demonstrates taht the prediction model accurately captures the direction of effect. 

#### 3. Evolutionary Constraint

Functionally important variants tend to be under evolutionary constraint. We calculated GERP score, which measures the number of substitutions "rejected" by evolutionary constraint, for top predicted variants and randomly sampled variants in 31 cell types. Higehr GERP score indicates greater magnitude of evolutionary constraint. As shown in fig3, for most cell types, our prediction model successfully prioritized genetic variants that are under higher evolutionary constraint and more likely to have siginificant biological functions. 

#### 4. Purifying Selection

Because more DNA changes are harmful than are beneficial, negative selection plays an important role in maintaining the long-term stability of biological structures by removing deleterious mutations. This is especially true for functionally important variants, whose change may disrupt essential biological functions. We obtained minor allele frequency from gnomAD database for all variants within peak regions of 31 chromatin profiles and plotted them against their predicted functional effects. As shown in fig4, variants with larger predicted functional effects tend to have lower minor allele frequency, which indicates the acting of purifying selection. 



## Figures

![
**ASoC Enrichment.**
Bar plot comparing the enrichment of allele specific open chromatin variants among three groups in two cell types.
](https://github.com/sq-96/resources/raw/master/ASoC%20Enrichment.png "Square image"){#fig:ASoC Enrichment}

![
**Cell Type Check.**
Bar plot comparing the enrichment of allele specific open chromatin variants among 31 groups in two cell types.
](https://github.com/sq-96/resources/raw/master/cell%20type%20check.png "Square image"){#fig:ASoC Enrichment}

![
**GERP Score Distribution.**
Bar plot comparing the evolutionary constraint between two groups in 31 cell types.
](https://github.com/sq-96/resources/raw/master/GERP_Score.png "Square image"){#fig:GERP Score Distribution}

![
**Sign Consistency.**
Scatter plot comparing the observed allelic imbalance and the predicted difference in functional effects between reference and alternative alleles.
](https://github.com/sq-96/resources/raw/master/sign%20consistency.png "Square image"){#fig:Sign Consistency}

![
**Minor Allele Frequency.**
Scatter plot showing the negtive correlation between minor allele frequency and Brain-ResNet predicted scores.
](https://github.com/sq-96/resources/raw/master/AF%20vs%20Score.png "Square image"){#fig:Minor Allele Frequency}

![
**Heatmap.**
Heatmap showing functional effects of credible set SNPs in 31 cell types.
](https://github.com/sq-96/resources/raw/master/heatmap_col_norm.png "Square image"){#fig:Heatmap}

![
**Tracks.**
Scatter plot showing pvalue, pip and functional effects of the candidate SNP.
](https://github.com/sq-96/resources/raw/master/rs13261217_CSMD1.png "Square image"){#fig:Heatmap}

