#### 1. Enrichment of ASoC Variants
All single nucleotide variants (SNVs) within functional regions (open chromatin or H3K27ac) in each cell type are retrieved from 1000 Genomes Project. SNVs are then ranked by MetaChrom score, Funsig score and CADD score in descending order and the top 10,000 variants are selected as potential functional variants. In each functional regions, we also randomly sampled 100,000 variants as control variants. Funsig score is from the DeepSEA Server and CADD score is from annovar. ASC variants in NPC and Glut are obtained from a neuron ATAC-seq paper. We count the number of ASC variants in potential functional variants and control variants, and Enrichment of ASC variants is calculated by fisher exact test. 

In the neuron ATAC-seq study, iPSCs of 20 individuals are differentiated into neural progenitor cells (NPC) and gutamatergic (iN-Glut) neurons. ATAC-seq is perform in each cell type, and 5,611 and 3,547 ASoC SNPs are identified in NPC and iN-Glut cells by allelic imbalance test, respectively.

#### 2. Sign Consistentcy
For ASC variants, we define its observed allelic imbalance as log(ref reads/alt reads) and predicted allelic imbalace as log(ref pred/alt pred). Correlation between observed allelic imbalance and predicted allelic imbalace is calcualted by Spearman's rank correlation coefficient.

#### 3. Evolutionary Constraint
GERP scores of potential functional variants and control variants are generated by Annovar. P value is calculated by the Wilcoxon Rank-Sum Test.


#### 4. Purifying Selection
Minor allele frequency of potential functional variants and control variants is obtained from gnomAD. The MetaChrom score ranges from 0 to 1. We splitted variants into 5 bins according to the MetaChrom score, We created five bins, namely 0-0.05, 0.05-0.1, 0.1-0.15, 0.15-0.2 and 0.2-1.0, and split variants into bins according their MetaChrom score. In each bin, We calculated the mean of minor allele frequency and standard error of the mean.
